v2/concordium/service.proto

```diff
@@ -140,6 +140,54 @@ service Queries {
   // non-finalized transaction hashes have been returned.
   rpc GetAccountNonFinalizedTransactions (AccountAddress) returns (stream TransactionHash);
 
+  // Shut down the node.
+  // Return a GRPC error if the shutdown failed.
+  rpc Shutdown(Empty) returns (Empty);
+  
+  // Suggest to a peer to connect to the submitted peer details.
+  // This, if successful, adds the peer to the list of given addresses.
+  // Otherwise return a GRPC error.
+  rpc PeerConnect (IpSocketAddress) returns (Empty);
+
+  // Disconnect from the peer and remove them from the given addresses list
+  // if they are on it. Return if the request was processed successfully.
+  // Otherwise return a GRPC error.
```

Comment:
There is a slight ambiguity here. What if the peer is not connected yet, is this an "OK", or error?